src/components/AsideHeader/__stories__/AsideHeaderShowcase.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {FC, useState} from 'react';
+import React, {FC} from 'react';
```

Comment:
Maybe then we’ll remove `FC` import too?

Reply:
fixed